src/main/java/com/resource/TwitterWorks.java

```diff
@@ -0,0 +1,60 @@
+package com.resource;
+
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.TwitterException;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/api/1.0/twitter")
+public class TwitterWorks {
```

Comment:
Change the class name please

Reply:
i tried to rename the file but not succeed so i created a file SendRetrieveTweets.java and added all TwitterWorks.java content in it.  deleted  TwitterWorks.java